client/scala/src/main/scala/ai/verta/blobs/dataset/Dataset.scala

```diff
@@ -38,9 +50,14 @@ trait Dataset extends Blob {
     val shared = contents.keySet.intersect(other.contents.keySet)
     contents.filterKeys(shared).equals(other.contents.filterKeys(shared))
   }
+
+  /** Clean up the uploaded components */
+  protected def cleanUpUploadedComponents(uploadMap: Map[String, UploadComponent]): Try[Unit] = Success(())
 }
 
 object Dataset {
+  private val BufferSize = 8192
```

Comment:
We should also increase this buffer for the same reason. I missed it before. I also generally recommend placing the constant close to where they're used.

Reply:
Fixed.